electron_src/main.js

```diff
@@ -140,12 +140,6 @@ function createWindow() {
       windowState.height = height;
     }
   });
-  mainWindow.on('maximize', () => {
```

Comment:
i'm not sure why delete this?

Reply:
There is double operation on windowState both in ipc control and an event listener. In the previous commit, I removed the event listener, which is not correct since other part of the code also uses maximize, so I remove the operation on windowState during ipc Control in my latest commit.